pages/projects.js

```diff
@@ -0,0 +1,357 @@
+import * as React from 'react';
+import { useRouter } from 'next/router';
+
+import PropTypes from 'prop-types';
```

Comment:
Would it be cleaner to destructure items in one import from material-ui/core? Is there a downside to the approach of destructuring vs one line importing?

Reply:
done